wca-guidelines.md

```diff
@@ -128,7 +128,9 @@ To be more informative, each Guideline is classified using one of the following
 - 9b3b+) [CLARIFICATION] The "Mean of 3" statistic does not affect the competitor's ranking in a "Best of 3" round (which is based on the best result). The WCA recognizes it outside of the competition format.
 - 9f1+) [EXAMPLE] If the timer displays 12.678 for an attempt, the original recorded time is 12.67 (drop any digit after a hundredth of a second). A judge may write the full-precision time on the score sheet (and keep the precision in the calculation of penalties), as long as the score taker truncates it for the official results.
 - 9f4+) [EXPLANATION] The result of an attempt is DNF only if the competitor began the attempt by indicating that they are ready (see [Regulation A3b2](regulations:regulation:A3b2)) but the attempt was disqualified.
-- 9f5+) [CLARIFICATION] The result for an attempt is DNS if the competitor was eligible for the attempt and did not start it (see [Regulation A3b2](regulations:regulation:A3b2)). If the competitor did not qualify for an attempt (e.g. in a combined round), they have no result for the attempt.
+- 9f5+) [CLARIFICATION] The result for an attempt is DNS if the competitor was eligible for the attempt and did not start it (see [Regulation A3b2](regulations:regulation:A3b2)). If the competitor did not qualify for an attempt (e.g. see [Guideline 9g+](guidelines:guideline:9g+)), they have no result for the attempt.
+- 9g+) [CLARIFICATION] A competitor who does not meet the cutoff result during the cutoff phase solves has no results for the remaining solves (rather than a result of DNS for each of the remaining solves).
```

Comment:
I think it's strange to use the phrase "cutoff result" here, where in [regulation 9g](https://www.worldcubeassociation.org/regulations/#9g) we say "designated cutoff".

Reply:
nit: I'm not sure the word "solves" has any value in the phrase "cutoff phase solves". I'd consider replacing that with simply "cutoff phase".